### PR TITLE
Auto Gen workflow

### PR DESCRIPTION
**Pull request by MosheBlumbergX**
I've added an auto gen workflow outlining how to use the auto gen certificates

### CODE REVIEW

File: security/production-secure-deploy-auto-gen-certs/README.rst [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+
+  - SASL/Plain or mTLS
+
+  - For SASL/Plain, the identity can come from LDAP server
+
+- Enable Confluent Role Based Access Control for authorization, with user/group identity coming from LDAP server
+
+- Enable TLS for network encryption - both internal (between CP components) and external (Clients to CP components)
+
+In this deployment scenario, we will set this up, choosing SASL/Plain for authentication, using user provided CA certificate, CFK will auto generate the components certificates for you.
```
Comment (rohit2b) [inline security/production-secure-deploy-auto-gen-certs/README.rst:16]: Suggest changing this text to:

In this deployment scenario, we'll choose SASL/Plain for authentication and configure TLS encryption using CFK auto-generated component certificates. 
You'll need to provide a certificate authority certificate for CFK to auto-generate the component certificates.

File: security/production-secure-deploy-auto-gen-certs/README.rst [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+You'll use Kubernetes secrets to provide credential configurations.
+
+With Kubernetes secrets, credential management (defining, configuring, updating)
+can be done outside of the Confluent For Kubernetes. You define the configuration
+secret, and then tell Confluent For Kubernetes where to find the configuration.
+   
+To support the above deployment scenario, you need to provide the following
+credentials:
+
+* CA private key and TLS Certificates
```
Comment (rohit2b) [inline security/production-secure-deploy-auto-gen-certs/README.rst:106]: Change text to:

* Certificate authority - private key and certificate

**Review by rohit2b — CHANGES_REQUESTED** [review]
Suggested two changes, apart from that looks good!

**Comment by MosheBlumbergX** [comment]
https://github.com/confluentinc/confluent-kubernetes-examples/pull/43 new pull

**Pull request closed without merging**